File: api/Entities/Dice.cs
namespace api.Entities
{
  public class Dice
  {
	  public int DiceId { get; set; }
	  public int Count { get; set; }
	  public int Sides { get; set; }

	  public int AttackId { get; set; }
	  public Attack AttackRoll { get; set; }
  }
}

File: src/app/common/dice.ts
export default class Dice {
	public Count: number;
	public Sides: number;

	constructor(count: number, sides: number) {
		this.Count = count;
		this.Sides = sides;
	}

	toString() {
		return `${this.Count}d${this.Sides}`;
	}
}
